docs/manuals/source/Appendix/ReleaseNotes.rst

```diff
@@ -24,13 +24,197 @@ This chapter concentrates on things to do when updating an existing Bareos insta
+   * - **URL**
+     - https://download.bareos.com/bareos/release/19.2/
+..
+   * - **Release Ticket**
+     - :issue:`0000`
+..
+   * - **LOC**
+     - 123456+ 12345-
+
+.. csv-table:: binary package availablility in the `bareos.com subscription repos <https://www.bareos.com/en/Subscription.html>`_
+   :header: "Distribution", "Architecture"
+   :widths: auto
+
+   CentOS_6, "x86_64"
+   CentOS_7, "x86_64"
+   Debian_8.0, "i586,x86_64"
+   Debian_9.0, "i586,x86_64"
+   Fedora_28, "x86_64"
+   Fedora_29, "x86_64"
+   FreeBSD_11.2, "x86_64"
+   MacOS, "x86_64"
+   RHEL_6, "x86_64"
+   RHEL_7, "x86_64"
+   SLE_12_SP3, "x86_64"
+   SLE_12_SP4, "x86_64"
+   SLE_15, "x86_64"
+   openSUSE_Leap_15.0, "x86_64"
+   Univention_4.3, "x86_64"
+   Windows, "32Bit, 64Bit"
+   xUbuntu_14.04, "i586,x86_64"
+   xUbuntu_16.04, "i586,x86_64"
+   xUbuntu_18.04, "x86_64"
+
+New Features
+^^^^^^^^^^^^
+* |webui|: :ref:`section-webui-restore` a specific file version
+* Display a summary before performing a restore in the |webui|
+* New configuration parameter *filetree_refresh_timeout* has been added to configuration.ini file of the |webui| to set a custom timeout while refreshing the file tree (.bvfs_cache_update) in the restore module (:ref:`section-updating-bvfs-cache-frequently`)
+* Configuration: :ref:`DirectorResourceUser` for pam authentication
+* Configuration: : Simplified :ref:`StorageResourceMultipliedDevice` Storage Configuration
+
+Changed Features
+^^^^^^^^^^^^^^^^
+* Improved Command ACL handling and error messages in the |webui|
+* Restricted console profile examples for limited and read-only purposes have been added to the |webui| package
+* Updated |webui| localization
+* Configuration: Do not allow empty passwords for mandatory passwords
+* All deamons: Generate a log message if extended attributes are disabled automatically
+* All deamons: replace crc32 implementation with a faster one
+* |dir|: Add support for MySQL 8
+
+Deprecated and Removed Features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* |webui|: Functionality of pre-selecting a backup client in the restore module has been removed
+
+Bugs Fixed
+^^^^^^^^^^
+* :ticket:`385`: bareos daemon stop restart hang if bareos-tray-monitor is connected
+* :ticket:`975`: .bvfs\_lsdirs limit offset command parameters do not work properly
+* :ticket:`1004`: Very high cpu usage on Debian stretch
+* :ticket:`1034`: Read error on tape may be misinterpreted as end-of-tape
+* :ticket:`1045`: webui login problem
+* :ticket:`1049`: Translation in German webUI
+* :ticket:`1064`: bconsole crashes after 5 minutes in restore operation due to forgotten watchdog
+* :ticket:`1073`: pthread\_detach for FreeBSD (PR169)
+* :ticket:`1091`: NDMP to NDMP Copy Job Fails
+* :ticket:`1123`: Director can crash during TwoWay Authentication
+* Fix Application Controller Plugin - CommandACLPlugin
+* Fix overflowing top navigation bar content hiding tab navigation in some modules in the |webui|
+* Fix buffer overrun in function PathAppend
```

Comment:
This is also mentioned in bareos-18.2.7 release notes.

Reply:
Your source was probably outdated, pls pull again.

Reply:
found it